driver/docker/driver.go

```diff
@@ -144,6 +135,20 @@ func (d *Driver) IsMobyDriver() bool {
 	return true
 }
 
+func (d *Driver) UsesContainerdSnapshotter(ctx context.Context) bool {
```

Comment:
Can you just define a new `Feature` for this. Seems lot of duplication and extra unnecessary requests.